setup.py

```diff
@@ -19,8 +19,8 @@
       py_modules=['Chandra.axTime3', 'Chandra.Time'],
       version='3.20',
       zip_safe=False,
-      test_suite="Chandra.test_Time",
-      packages=['Chandra'],
-      package_dir={'Chandra': 'Chandra'},
+      test_suite="Chandra.Time.test_Time",
+      packages=['Chandra', 'Chandra/Time'],
```

Comment:
Oh.  I hadn't moved axTime3 and hadn't originally planned to. 
What does Chandra.Time.tests do in this context?